tests/foreman/api/test_computeresource_vmware.py

```diff
@@ -79,11 +79,20 @@ def test_positive_provision_end_to_end(
             'cluster': f'{settings.vmware.cluster}',
             'start': '1',
             'guest_id': 'rhel8_64Guest',
+            'scsi_controllers': [{"type": "ParaVirtualSCSIController", "key": 1001}],
+            'nvme_controllers': [{"type": "VirtualNVMEController", "key": 2001}],
             'volumes_attributes': {
                 '0': {
-                    'size_gb': 10,
+                    'size_gb': 8,
                     'thin': '1',
                     'storage_pod': f'{settings.vmware.datastore_cluster}',
+                    'controller_key': 2001,
```

Comment:
is this some random key we specifying?

Reply:
Yes any random key, but has to be matched with any of the controller specified above